content/news/2022-04-12-bevy-0.7/index.md

```diff
@@ -0,0 +1,383 @@
+youtube = "cartdev"
+image = "FIXME.png"
+show_image = true
+image_subtitle = ""
+image_subtitle_link = ""
++++
+
+Thanks to **X** contributors, **X** pull requests, and our [**generous sponsors**](https://github.com/sponsors/cart), I'm happy to announce the **Bevy 0.7** release on [crates.io](https://crates.io/crates/bevy)!
+
+For those who don't know, Bevy is a refreshingly simple data-driven game engine built in Rust. You can check out [Quick Start Guide](/learn/book/getting-started/) to get started. Bevy is also free and open source forever! You can grab the full [source code](https://github.com/bevyengine/bevy) on GitHub. Check out [Bevy Assets](https://bevyengine.org/assets) for a collection of community-developed plugins, games, and learning resources.
+
+To update an existing Bevy App or Plugin to **Bevy 0.7**, check out our [0.6 to 0.7 Migration Guide](/learn/book/migration-guides/0.6-0.7/).
+
+As always, there are a _ton_ of new features, bug fixes, and quality of life tweaks in this release, but here are some of the highlights: 
+
+* Skeletal animation and mesh skinning
+* GLTF animation importing
+* Unlimited* point lights in a scene
+* Improved clustered forward rendering: dynamic/adaptive clustering, faster and more accurate cluster assignment
+* Compressed texture support (KTX2 / DDS / .basis): load more textures in a scene, faster
+* Compute shader / pipeline specialization: Bevy's flexible shader system was ported to compute shaders, enabling hot reloading, shader defs, and shader imports
+* Render to texture: cameras can now be configured to render to a texture instead of a window
+* Flexible user-customizable mesh vertex layouts in shaders
+* ECS improvements: Order systems using their names, Query::get_many, use conflicting parameters in systems via ParamSets, WorldQuery derives
+* More audio control: pause, volume, speed, and looping
+* Hot reloading for "plugin-provided / built-in assets"
+* Power usage options to enable only updating Bevy Apps when input occurs 
+
+<!-- more -->
+
+## Skeletal Animation
+
+<div class="release-feature-authors">authors: @james7132, @mockersf, @lassade, @Looooong</div>
+
+Bevy finally supports 3D skeletal animation!
+
+<video controls loop><source  src="skeletal_animation.mp4" type="video/mp4"/></video>
+
+<div style="font-size: 1.0rem" class="release-feature-authors">Scene Credits: <a href="https://skfb.ly/6TsvL">Tanabata evening - Kyoto inspired city scene</a> by Mathias Tossens is licensed under <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution</a>. Character model and animation are royalty free assets from Mixamo. 
+</div>
+
+Skeletal animations can now be played, paused, scrubbed, looped, reversed, and speed controlled using the new [`AnimationPlayer`] component and [`AnimationClip`] asset:
+
+```rust
+#[derive(Component)]
+struct Animations {
+    dance: Handle<AnimationClip>,
+}
+
+fn system(mut query: Query<(&Animations, &mut AnimationPlayer)>) {
```

Comment:
Yeah, I agree with DJ here (sorry Ida!). It's better to model good habits, and the added context of a name can help the reader clue in faster to what the example is supposed to do.

Reply:
Yeah. 
The version for describing the useless system was attempting to be a joke, but I do think that `fn system` is not a good pattern, since system names should always be verbs.

Reply:
Solid points. I wanted to make it clear that its a system, but modeling good behavior is more important. I'm convinced!